app/app/calculations/consts.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import math
+
+mm = 1.0e-3         # 0.001
+cm = 1.0e-2
+m = 1.0
+mm2 = 1.0e-6        # 0.000001
+cm2 = 1.0e-4
+kN = kNm = 1.0e3    # rowniez jako kN/m
+MPa = 1.0e6         # 1000000
+GPa = 1.0e9
+
+pi = math.pi
+sin = lambda x: math.sin(x)
+cos = lambda x: math.cos(x)
+ceil = lambda x: math.ceil(x)
```

Comment:
Czemu tu jest tak zrobione? Nie rozumiem tej idei, ja bym po prostu skorzystała z funkcji bezpośrednio w punching.py wywołując je, wszak po to są.